omod/src/main/webapp/resources/htmlforms/dm_htn_initial.xml

```diff
@@ -801,8 +885,9 @@
             <table>
                 <tbody>
                     <tr>
-                        <td>Provider Name________________________</td>
-                        <td>Designation__________________________</td>
+                        <td>Provider Name:<obs conceptId="1473"/></td>
+                        <td>Designation:Provider Name<obs conceptId="163556" answerConceptIds="164222,164223" answerLabels="Attending surgeon,Surgion Supervisor" style="radio"   answerSeparator=" tt " 
+                         /></td>
                         <td>Signature____________________________</td>
```

Comment:
This might NOT be available for an electronic form, unless someone has a scanned signature they can just paste here